ios/Exponent/Supporting/Info.plist

```diff
@@ -113,6 +113,8 @@
 	<true/>
 	<key>UIBackgroundModes</key>
 	<array>
+		<string>bluetooth-central</string>
```

Comment:
Should remove these if we aren't supporting the feature

Reply:
Let's remove these for now, and instead have a guide with the unimodule explaining what plist keys need to be set.